app/views/directives/edit-environment-variables.html

```diff
@@ -1,7 +1,7 @@
 <form ng-if="$ctrl.apiObject" name="$ctrl.form" class="mar-bottom-xl">
   <confirm-on-exit ng-if="$ctrl.canIUpdate && !$ctrl.ngReadonly" dirty="$ctrl.form.$dirty"></confirm-on-exit>
   <div ng-repeat="container in $ctrl.containers">
-    <h3>Container {{container.name}}</h3>
+    <h2>Container {{container.name}}</h2>
```

Comment:
I know this change was suggested by @jennyhaines, but I'm a little concerned that we're not making it on other browse pages / other tabs. All of the other headers are `h3` on the browse pages.

We might hold off on this and look at it in the large context of all the pages later. Any objections @jennyhaines ?

If so, we'd probably keep the "Environment From" header an `h4` (without the `section-label` class) keep the hierarchy.